go/prometheusexporter/webserver/webserver.go

```diff
@@ -38,19 +41,36 @@ func serveAliveness(w http.ResponseWriter, r *http.Request) {
 }
 
 func serveAppMetrics(w http.ResponseWriter, r *http.Request) {
-	vars := mux.Vars(r)
-	app := vars["application"]
-	env := vars["environment"]
+	app, env := getAppEnv(r)
+	h := collectormanager.GetRequestHandler(app + "-" + env)
+	serve(w, r, h.ServeHTTP)
+}
+
+func serveExceptionsMetrics(w http.ResponseWriter, r *http.Request) {
+	app, env := getAppEnv(r)
+	h := collectormanager.GetCollector(app + "-" + env)
```

Comment:
This looks strange to me, as it differs from the call to collectormanager.GetRequestHandler(app + "-" + env) above.